Blocks/URL/test.cc

```diff
@@ -447,3 +447,63 @@ TEST(URLTest, FillsCurrentStructsFromURLParameters) {
     }
   }
 }
+
+TEST(URLTest, GetURLPathSeparator) {
+  EXPECT_EQ('/', current::url::GetURLPathSeparator());
```

Comment:
I think checking the value of static constant isn't worth a separate test.. If we indeed need such kind of checks, why not make it as part of, for example, `JoinURLPath` test?